### PR TITLE
[decoupled-execution] refine buffer-manager diagram

### PR DESCRIPTION
**Pull request by yuxiamit**
Add an example state of the roots pointing to (LedgerInfo) buffer items.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/diem/diem/tree/main/developers.diem.com, and link to your PR here.)

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for V1 launch.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

File: consensus/src/experimental/buffer_manager.rs [hunk 1]
```diff
@@ -39,21 +39,28 @@ pub struct OrderedBlocks {
 }
 
 /*
- *                       LedgerInfo Buffer
- *
- *                      ┌────────────┐                    ┌────────────┐
+ *                                                         ┌─────────┐
+ *                                                         │ Signing │
+ *                                                         │ Root    │
+ *                                                         └────┬────┘
```
Comment (zekun000) [inline consensus/src/experimental/buffer_manager.rs:45]: I thought signing root is always the aggregation root?

**Review by zekun000 — APPROVED** [review]
I guess decoupling them doesn't hurt and may improve the performance.

File: consensus/src/experimental/buffer_manager.rs (hunk 1 above)
Comment (yuxiamit) [reply consensus/src/experimental/buffer_manager.rs:45]: I intend to allow concurrent signature aggregation (as long as the target ledger info enters the buffer, we should keep the signature received from the commit vote).

**Comment by yuxiamit** [comment]
> I guess decoupling them doesn't hurt and may improve the performance.

I'm working on refactoring the buffer item to make it less confusing.

**Pull request closed without merging**